tartufo/commands/scan_local_repo.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Optional, Tuple
+
+import click
+
+from tartufo import types, util
+from tartufo.scanner import GitRepoScanner, Issue
+
+
+@click.command("scan-local-repo")
+@click.option("--since-commit", help="Only scan from a given commit hash.")
+@click.option(
+    "--max-depth",
+    default=1000000,
+    help="The max commit depth to go back when searching for secrets."
+    " [default: 1000000]",
```

Comment:
very minor: would it be better to have the help print the default value rather than having a second hard coded entry?

Reply:
I agree, that would be ideal. I think that might require a bit of a deeper dive into and customization of `click`, however. The help text is currently entirely auto-generated by `click`. Sounds to me like a good feature request. 😄

Reply:
Upon further investigation, this is actually not hard at all! https://click.palletsprojects.com/en/7.x/api/#click.Option

> show_default – controls if the default value should be shown on the help page. Normally, defaults are not shown. If this value is a string, it shows the string instead of the value. This is particularly useful for dynamic options.

I will make it so, for all the things!